tests/__helpers__/task.ts

```diff
@@ -12,3 +12,39 @@ export const taskData = {
   creationTimestamp: new Date('2021-04-22T14:29:45.586Z'),
   updateTimestamp: new Date('2021-04-22T14:29:45.586Z'),
 };
+
+export interface InputType {
+  propertyOne: string;
+  propertyTwo: string;
+  propertyThree: string;
+}
+
+export interface OutputType {
+  propertyOne: string;
```

Comment:
silly: to avoid confusion it might be nice make them InputType and OutputType different